pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -766,6 +767,18 @@ func taskWorkspaceByWorkspaceVolumeSource(wb v1alpha1.WorkspaceBinding, taskWork
 	return binding
 }
 
+// combinedSubPath returns the combined value of the optional subPath from workspaceBinding and the optional
+// subPath from pipelineTask. If both is set, they are joined with a slash.
+func combinedSubPath(workspaceSubPath string, pipelineTaskSubPath string) string {
+	if workspaceSubPath == "" {
+		return pipelineTaskSubPath
+	} else if pipelineTaskSubPath == "" {
+		return workspaceSubPath
+	} else {
+		return fmt.Sprintf("%s/%s", workspaceSubPath, pipelineTaskSubPath)
```

Comment:
suggest [`filepath.Join` here](https://golang.org/pkg/path/filepath/#Join), which will also clean the paths in case extra slashes get included at the end / beginning of either the strings.

Reply:
Thanks, sounds like a good suggestion! I didn't know about it.